From ecee76aa08709d18bf93a3a972617c99e7e68c78 Mon Sep 17 00:00:00 2001
From: Thao Nguyen <thao.nguyen.yb@rvc.renesas.com>
Date: Tue, 23 Feb 2016 12:07:16 +0700
Subject: [PATCH 1285/2066] thermal: rcar_gen3_thermal: Use linear equation as
 default formula

Signed-off-by: Thao Nguyen <thao.nguyen.yb@rvc.renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 drivers/thermal/rcar_gen3_thermal.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/thermal/rcar_gen3_thermal.c b/drivers/thermal/rcar_gen3_thermal.c
index e32ce82..8761644 100644
--- a/drivers/thermal/rcar_gen3_thermal.c
+++ b/drivers/thermal/rcar_gen3_thermal.c
@@ -72,8 +72,8 @@
  * Default is using quadratic equation.
  * To switch to linear formula calculation,
  * please comment out APPLY_QUADRATIC_EQUATION macro.
- */
-#define APPLY_QUADRATIC_EQUATION
+*/
+/* #define APPLY_QUADRATIC_EQUATION */
 
 #ifdef APPLY_QUADRATIC_EQUATION
 /* This struct is for quadratic equation.
-- 
1.7.5.4

